specifyweb/frontend/js_src/lib/components/Atoms/__tests__/DataEntry.test.ts

```diff
@@ -35,29 +35,32 @@ describe('DataEntry.Cell', () => {
       children: 'Test',
       colSpan: 1,
       align: 'right',
+      verticalAlign: 'stretch',
       visible: true,
     },
-    'colspan 1, align right'
+    'colspan 1, align right, self stretch'
```

Comment:
Nice! I like that you didn't forget to update the tests
Though, for best coverage, could you make each of these tests use a different value of verticalAlign?